content/medo_manual/inloggen_en_veiligheid.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Inloggen en veiligheid
+sort: 100
+---
+
+* TOC
+{:toc}
+
+## Eerste gebruik
+
+U ontvangt van ons een email met de activatie-link wanneer u uitgenodigd bent
+om deel te nemen aan het beheer van een Medo project. Wanneer u op deze link
+klikt, moet u uw naam invullen en een nieuwe wachtwoord kiezen. Hierna wordt
+u doorverwezen naar een scherm waar u een
+[Two-Factor Authentication](#two-factor-authentication) kunt instellen.
+
+## Two-Factor Authentication
+
+In Medo maken we gebruik van Two-Factor Authentication - een authenticatie vorm
```

Comment:
Medo maakt gebruik van twee-staps-verificatie

Probeer geen "we" te gebruiken in een handleiding